src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -610,13 +612,17 @@ public File createApprovedDARDocument() throws IOException {
             for (Election election : elections) {
                 Document dar = getDataAccessRequestByReferenceIdAsDocument(election.getReferenceId());
                 DataAccessRequest dataAccessRequest = findByReferenceId(election.getReferenceId());
+                User user = userDAO.findUserById(dataAccessRequest.getUserId());
                 try {
-                    if (dar != null) {
+                    if (dar != null && user != null) {
                         Integer datasetId = dataAccessRequest.getData().getDatasetIds().get(0);
                         String consentId = dataSetDAO.getAssociatedConsentIdByDataSetId(datasetId);
                         Consent consent = consentDAO.findConsentById(consentId);
-                        String profileName = userPropertyDAO.findPropertyValueByPK(dataAccessRequest.getUserId(), DarConstants.PROFILE_NAME);
-                        String institution = userPropertyDAO.findPropertyValueByPK(dataAccessRequest.getUserId(), DarConstants.INSTITUTION);
+                        String profileName = user.getDisplayName();
+                        if (user.getInstitutionId() == null) {
+                          logger.warn("No institution found for creator of this Data Access Request");
+                        }
+                        String institution = (user.getInstitutionId() == null) ? "" : institutionDAO.findInstitutionById(user.getInstitutionId()).getName();
```

Comment:
Prefer `Objects.isNull()` over direct null checks.
```suggestion
                        String institution = Objects.isNull(user.getInstitutionId()) ? "" : institutionDAO.findInstitutionById(user.getInstitutionId()).getName();
```